daemon/mgr/container.go

```diff
@@ -595,6 +607,53 @@ func (mgr *ContainerManager) createContainerdContainer(ctx context.Context, c *C
 	return c.Write(mgr.Store)
 }
 
+func (mgr *ContainerManager) ensureRootFSMounted(rootfs string, snapData map[string]string) error {
+	if rootfs == "" || snapData == nil {
```

Comment:
If it is better to use `len(snapData) == 0`? @HusterWan 
Since `snapData` has possibility to be `map[string]string{}`.

Reply:
Agree :)